<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13215" started-at="2017-08-17T09:02:22Z" finished-at="2017-08-17T09:02:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13215" started-at="2017-08-17T09:02:22Z" finished-at="2017-08-17T09:02:35Z">
      <class name="Salle.TestClasses.SalleHeader">
        <test-method status="PASS" signature="AntesHeader()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="AntesHeader" is-config="true" duration-ms="6041" started-at="2017-08-17T11:02:22Z" finished-at="2017-08-17T11:02:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AntesHeader -->
        <test-method status="PASS" signature="NuevosAlumnos()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="NuevosAlumnos" duration-ms="1340" started-at="2017-08-17T11:02:28Z" finished-at="2017-08-17T11:02:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NuevosAlumnos -->
        <test-method status="PASS" signature="Alumnos()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="Alumnos" duration-ms="1802" started-at="2017-08-17T11:02:29Z" finished-at="2017-08-17T11:02:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Alumnos -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-08-17T11:02:29Z" finished-at="2017-08-17T11:02:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NuevosAlumnos status=SUCCESS method=SalleHeader.NuevosAlumnos()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-08-17T11:02:31Z" finished-at="2017-08-17T11:02:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Alumnos status=SUCCESS method=SalleHeader.Alumnos()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="Alumni()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="Alumni" duration-ms="1290" started-at="2017-08-17T11:02:31Z" finished-at="2017-08-17T11:02:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Alumni -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-08-17T11:02:33Z" finished-at="2017-08-17T11:02:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Alumni status=SUCCESS method=SalleHeader.Alumni()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="Empresas()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="Empresas" duration-ms="695" started-at="2017-08-17T11:02:33Z" finished-at="2017-08-17T11:02:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Salle.TestClasses.SalleHeader.Empresas(SalleHeader.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Empresas -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="tearDown" is-config="true" duration-ms="647" started-at="2017-08-17T11:02:33Z" finished-at="2017-08-17T11:02:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Empresas status=FAILURE method=SalleHeader.Empresas()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="DespuesHeader()[pri:0, instance:Salle.TestClasses.SalleHeader@58cbafc2]" name="DespuesHeader" is-config="true" duration-ms="1365" started-at="2017-08-17T11:02:34Z" finished-at="2017-08-17T11:02:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DespuesHeader -->
      </class> <!-- Salle.TestClasses.SalleHeader -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
